  NetworkProcess_UDP.cpp
NetworkProcess_UDP.obj : error LNK2019: closesocket 외부 기호(참조 위치: "public: __cdecl NetWorkProcess_UDP::NetWorkProcess_UDP(void)" (??0NetWorkProcess_UDP@@QEAA@XZ) 함수)에서 확인하지 못했습니다.
NetworkProcess_UDP.obj : error LNK2019: htons 외부 기호(참조 위치: "public: __cdecl NetWorkProcess_UDP::NetWorkProcess_UDP(void)" (??0NetWorkProcess_UDP@@QEAA@XZ) 함수)에서 확인하지 못했습니다.
NetworkProcess_UDP.obj : error LNK2019: inet_addr 외부 기호(참조 위치: "public: __cdecl NetWorkProcess_UDP::NetWorkProcess_UDP(void)" (??0NetWorkProcess_UDP@@QEAA@XZ) 함수)에서 확인하지 못했습니다.
NetworkProcess_UDP.obj : error LNK2019: sendto 외부 기호(참조 위치: "public: void __cdecl NetWorkProcess_UDP::SendPacket(wchar_t *)" (?SendPacket@NetWorkProcess_UDP@@QEAAXPEA_W@Z) 함수)에서 확인하지 못했습니다.
NetworkProcess_UDP.obj : error LNK2019: socket 외부 기호(참조 위치: "public: __cdecl NetWorkProcess_UDP::NetWorkProcess_UDP(void)" (??0NetWorkProcess_UDP@@QEAA@XZ) 함수)에서 확인하지 못했습니다.
NetworkProcess_UDP.obj : error LNK2019: WSAStartup 외부 기호(참조 위치: "public: __cdecl NetWorkProcess_UDP::NetWorkProcess_UDP(void)" (??0NetWorkProcess_UDP@@QEAA@XZ) 함수)에서 확인하지 못했습니다.
NetworkProcess_UDP.obj : error LNK2019: WSACleanup 외부 기호(참조 위치: "public: __cdecl NetWorkProcess_UDP::NetWorkProcess_UDP(void)" (??0NetWorkProcess_UDP@@QEAA@XZ) 함수)에서 확인하지 못했습니다.
C:\Users\KimChanYong\Documents\OMOC\x64\Debug\OMOC.exe : fatal error LNK1120: 7개의 확인할 수 없는 외부 참조입니다.
